esti/main_test.go

```diff
@@ -54,7 +54,8 @@ var (
 func TestMain(m *testing.M) {
 	systemTests := flag.Bool("system-tests", false, "Run system tests")
 	useLocalCredentials := flag.Bool("use-local-credentials", false, "Generate local API key during `lakefs setup'")
-
+	adminAccessKeyID := flag.String("admin-access-key-id", "AKIAIOSFDNN7EXAMPLEQ", "lakeFS Admin access key ID")
```

Comment:
Can you define the default access key id and secret key as consts in the package

Reply:
I agree that it makes sense to set them as const, but I'm afraid that people will make confused and use those consts in the tests instead of `params.AdminAccessKeyID` and `params.AdminSecretAccessKey`. WDYT?

Reply:
The reason I suggested it was so that we can run tests using `use-local-credentials` and no provide the keys.
Then we could use the const defaults and not hardcode the credentials in the test itself